apps/account-worker/src/transaction_publisher/publisher.service.ts

```diff
@@ -196,6 +204,34 @@ export class TransactionPublisherService extends BaseConsumer implements OnAppli
     }
   }
 
+  async processProxyTxn(
+    ext: any,
```

Comment:
Is it possible to use some more specific type information here instead of `any`?